test/schedule/data_test.exs

```diff
@@ -165,40 +230,46 @@ defmodule Schedule.DataTest do
         }
       }
 
-      trips = Data.trips_by_id(data, ["t1", "t2"])
+      Data.save_schedule_data_to_tables(tables, data)
+
+      trips = Data.trips_by_id(tables, ["t1", "t2"])
       assert Enum.count(trips) == 2
 
-      trip_ids = Enum.map(trips, fn {_trip_id, trip} -> trip.id end)
+      trip_ids = Map.keys(trips)
       assert "t1" in trip_ids
       assert "t2" in trip_ids
     end
   end
 
   describe "block" do
-    test "block returns the block" do
+    test "block returns the block", %{tables: tables} do
       block = build(:block)
 
       data = %Data{
-        blocks: %{{"block", "service"} => block}
+        blocks: %{{"schedule", "block"} => block}
```

Comment:
**praise:** good catch on the typo here, even if it didn't impact the semantics of the test at all it could definitely cause confusion.